### ISSUE TITLE
a11y - Asterisk for required fields disappears when rendered without css

### ISSUE THREAD

**Issue by LaurensBurger** [open-formulieren/open-forms#5390]
### Thema / Theme

Frontend

SED 120

### Omschrijving / Description

https://www.w3.org/TR/WCAG20-TECHS/F87.html 

Since the Asterisk is not decorative - the removal of CSS by the user should not remove the Asterisk's from required fields. 
The result of using a pseudo element for the asterisk in the case of users who need to render forms without styling - is a less accessible form because they cannot tell required from non-required fields.

**Comment by joeribekker** [comment]
@LaurensBurger so, you remove the css? or you add css that nullifies something? What do you expect us to do here?

**Comment by joeribekker** [comment]
Refinement: This never popped up in any accessibility audit but we can do it. Unfortunately it's not super trivial. Estimate: 3-4 days
